docs/src/app/components/pages/components/tabs.jsx

```diff
@@ -101,7 +101,7 @@ class TabsPage extends React.Component {
         infoArray: [
           {
             name: 'onChange',
-            type: 'function(e, tab)',
+            type: 'function(value, e, tab)',
             header: 'optional',
             desc: 'Fired on touch or tap of a tab. Passes the index and the tab element.'
```

Comment:
@Zadielerick This description in not correct since we're not passing the index anymore.